Buildings/Controls/OBC/CDL/Logical/Timer.mo

```diff
@@ -93,6 +110,11 @@ the timer will not fully stopped but hold the accumulated true input time.
 </html>", revisions="<html>
 <ul>
 <li>
+July 23, 2018, by Jianjun Hu:<br/>
+Added conditional boolean input for resetting output.  This is for
```

Comment:
maybe add `for cumulative time measuring.` Also there is an extra space after the sentence.